### ISSUE TITLE
Cannot uninstall plugin

### ISSUE THREAD

**Issue by zabojad** [getsentry/sentry-cordova#53]
It will get stuck when asking for the DSN:  `Missing secret in DSN`. Even if I give the correct full DSN...

**Comment by zabojad** [comment]
I finally used the old format DSN and it passed...

**Comment by zabojad** [comment]
The error was with the cordova cli when doing `cordova plugin rm sentry-cordova`. Then several prompts shows and one of them is the DSN. It refuses the new/current format of DSN and passes with the old format. I had both android and ios platforms installed in the project.

**Comment by HazAT** [comment]
OK, people reported this before.
If I call `cordova plugin rm sentry-cordova` I get this:
```
no URL provided, fallback to https://sentry.io/
Running Sentry Wizard...
version: 0.9.5 | sentry-cli version: 1.30.4

😢  Successfully removed Sentry from your project 😢
Uninstalling sentry-cordova from ios
Removing "sentry-cordova"
Removing plugin sentry-cordova from config.xml file...
Removing sentry-cordova from package.json
```

no prompts at all (and it should be like this).

**Comment by zabojad** [comment]
Wow... I have several prompts (organization name, project name, and finally DSN).

I've just done it again, here is the cordova cli output I get (after obfuscation of the organization and project names):
```
$ cordova plugin add sentry-cordova
Installing "sentry-cordova" for android
no URL provided, fallback to https://sentry.io/
Running Sentry Wizard...
version: 0.9.6 | sentry-cli version: 1.30.5
Sentry Wizard will you help to configure your project
Thank you for using Sentry :)
will not configure ios


Please open
https://sentry.io/account/settings/wizard/gtjimobkgqey1gmp5dw4joy74cyktv5uhrps9qawrxyyed5z9z7nrwetgdi7m0h0/
in your browser (if it's not open already)


? Please select your project in Sentry: my-org / MyProj
Successfully set up android for cordova

🎉  Successfully set up Sentry for your project 🎉
ANDROID_HOME=/Users/thomasfetiveau/Library/Android/sdk
JAVA_HOME=/Users/thomasfetiveau/java
Subproject Path: CordovaLib
Starting a Gradle Daemon (subsequent builds will be faster)

> Configure project :
The Task.leftShift(Closure) method has been deprecated and is scheduled to be removed in Gradle 5.0. Please use Task.doLast(Action) instead.
        at build_8b7sgkq3xfs7rri3aa42p91rj.run(/Users/thomasfetiveau/wksp_my-org/my-org-apps/cordova-my-proj/platforms/android/build.gradle:141)
Configuration 'compile' in project ':' is deprecated. Use 'implementation' instead.

> Configure project :CordovaLib
publishNonDefault is deprecated and has no effect anymore. All variants are now published.


BUILD SUCCESSFUL in 8s
2 actionable tasks: 1 executed, 1 up-to-date
Subproject Path: CordovaLib
Installing "sentry-cordova" for ios
no URL provided, fallback to https://sentry.io/
Running Sentry Wizard...
version: 0.9.6 | sentry-cli version: 1.30.5
will not configure android


Please open
https://sentry.io/account/settings/wizard/c4v2wpl1m1vtnvd1vpylh5hgqbw05alsomsx1skiorptwmjvboksnbouwe7lvm4d/
in your browser (if it's not open already)


? Please select your project in Sentry: my-org / MyProj
Successfully set up ios for cordova

🎉  Successfully set up Sentry for your project 🎉
Sentry: copying platforms/ios/sentry.properties
Adding sentry-cordova to package.json
Saved plugin info for "sentry-cordova" to config.xml
pc29:cordova-my-proj thomasfetiveau$ cordova plugin rm sentry-cordova
no URL provided, fallback to https://sentry.io/
Running Sentry Wizard...
version: 0.9.6 | sentry-cli version: 1.30.5
Please copy/paste your organization slug
It can be found in the url https://sentry.io/organization_slug/project_slug
? Organization Slug: my-org
Please copy/paste your project slug
It can be found in the url https://sentry.io/my-org/project_slug
? Project Slug: my-proj
Please copy/paste your DSN
It can be found here: https://sentry.io/my-org/my-proj
? DSN: https://4ac20c7272dd41a1b38fe723b8f49568:b78c4d2bce4d48fabb81550abc90ac80@sentry.io/1194997
Please copy/paste your auth token
It can be found here: https://sentry.io/api/
In case there is none yet, create one with [project:releases] permission
? Auth Token: 7999de0392cf4d638850e60fcf84823443a150c0bb73403f9599ad0591787419

😢  Successfully removed Sentry from your project 😢
Uninstalling sentry-cordova from android
Subproject Path: CordovaLib
no URL provided, fallback to https://sentry.io/
Running Sentry Wizard...
version: 0.9.6 | sentry-cli version: 1.30.5

😢  Successfully removed Sentry from your project 😢
Uninstalling sentry-cordova from ios
Removing "sentry-cordova"
Removing plugin sentry-cordova from config.xml file...
Removing sentry-cordova from package.json
pc29:cordova-my-proj thomasfetiveau$ 
```

**Comment by zabojad** [comment]
@HazAT obviously, we do not use the same versions. You have `version: 0.9.5 | sentry-cli version: 1.30.4` and I have `version: 0.9.6 | sentry-cli version: 1.30.5`. I don't know if this is the cause of the differences we have...

**Comment by HazAT** [comment]
Hey, this should be fixed in the new version `0.10.0`
Please update and do a 
`cordova plugin rm sentry-cordova` 
`cordova plugin add sentry-cordova`
